index.html

```diff
@@ -4,185 +4,222 @@
+			{ "method": ["Result", "unwrap"], "have": "Result<T,E>", "want": "T", "by": "propagating the result", "by_mod": null, "or": "panicking", "or_mod": "with a generic error message" },
+			{ "method": ["Result", "expect"], "have": "Result<T,E>", "want": "T", "by": "propagating the result", "by_mod": null, "or": "panicking", "or_mod": "with a specific error message" },
+			{ "method": ["Result", "unwrap_err"], "have": "Result<T,E>", "want": "E", "by": "panicking", "by_mod": "with a generic error message", "or": "propagating $ERRNAME", "or_mod": null },
+			{ "method": ["Result", "expect_err"], "have": "Result<T,E>", "want": "E", "by": "panicking", "by_mod": "with a specific error message", "or": "propagating $ERRNAME", "or_mod": null },
+			{ "method": ["Result", "transpose"], "have": "Result<Option<T>,E>", "want": "Option<Result<T,E>>", "by": "transposing the result", "by_mod": null, "or": "propagating $ERRNAME as the result", "or_mod": null },
+			{ "method": ["Iterator<Result>", "collect"], "have": "Iterator<Result<T,E>>", "want": "Result<Collection<T>,E>", "by": "collecting the results", "by_mod": null, "or": "propagating the first $ERRNAME", "or_mod": null },
+			{ "method": ["Iterator<Option>", "collect"], "have": "Iterator<Option<T>>", "want": "Option<Collection<T>>", "by": "collecting the results", "by_mod": null, "or": "propagating the first $ERRNAME", "or_mod": null },
+			{ "method": ["Iterator<Option>", "filter_map"], "have": "Iterator<Option<T>>", "want": "Iterator<T>", "by": "collecting the results", "by_mod": null, "or": "dropping $ERRNAME values", "or_mod": null },
+		];
+
+		var types = {
+			"Option": { "url": "https://doc.rust-lang.org/stable/std/option/enum.Option.html", "ERRNAME": "None" },
+			"Result": { "url": "https://doc.rust-lang.org/stable/std/result/enum.Result.html", "ERRNAME": "the error" },
+			"Iterator<Option>": { "url": "https://doc.rust-lang.org/stable/std/iter/trait.Iterator.html", "ERRNAME": "None" },
+			"Iterator<Result>": { "url": "https://doc.rust-lang.org/stable/std/iter/trait.Iterator.html", "ERRNAME": "error" },
+		};
+
+		function getColumn(key, constraints) {
+			var selected = methods.filter(function (el) {
+				for (c in constraints)
+					if (el[c] != constraints[c])
+						return false;
+				return true;
 			});
-			if (opts.length!=1 || matched) {
-				var opt=$('<option class="clear" value="">(clear)</option>');
-				if (!matched)
-					opt.attr('selected',true);
-				opts.unshift(opt);
+			return _(selected).map(function (row) {
+				return row[key];
+			}).sort().uniq(true).value();
+		}
+
+		function makeOption(text, errname, selected) {
+			var value = text;
+			var title = text.replace("$ERRNAME", errname, "g");
+
+			var opt = $('<option>').val(value).text(title);
+			if (value == selected)
+				opt.attr('selected', true);
+
+			opt.attr("title", title);
```

Comment:
You've increased the font size significantly such that the select boxes are now not wide enough to all fit next to each other which in turn leads to needing to add a tooltip here. It would be better if the text was never truncated.

Reply:
Can you point out a case where the text is truncated? I made sure that no text is truncated. The label is just for accessibility, happy to ditch it if you prefer so.

Reply:
Pushed a commit with a smaller font size.